src/app/accounts/accounts/accounts.component.ts

```diff
@@ -50,6 +54,10 @@ export class AccountsComponent implements OnInit {
     );
   }
 
+  ngOnDestroy() {
```

Comment:
it is good style, and it is logical  to position `ngOnDestroy` in the bottom of constructor